Khan/perseus
# [Locked Figure Labels] View locked point labels

## Summary:

- Add feature flag for locked point labels
- Add `labels` to LockedPointType
- Update builder
- Update stories and tests

Issue: https://khanacademy.atlassian.net/browse/LEMS-2346

## Test plan:
- Go to the following story: http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-point-labels-flag
- Confirm that there is a label "A" next to the locked point
- Got o the following story: http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-labels-flag
- Confirm there is no label "A" next to the locked point

<img width="484" alt="image" src="https://github.com/user-attachments/assets/cb1b90eb-3ff1-4fee-a91b-e255f8e0eae7">
